shared/components/map.js

```diff
@@ -324,7 +322,7 @@ export default class Map extends Component {
               this.setState({markerSelect: item.key});
             }}
           >
-            {(this.state.markerSelect === item.key) ?
+            {(Platform.OS === 'ios' && this.state.markerSelect === item.key) ?

```

Comment:
so until figuring out this issue, it would be hard to render numbers on clicked marker